crates/subspace-runtime/src/lib.rs

```diff
@@ -712,16 +714,16 @@ fn extract_feeds_block_object_mapping(
     base_offset: u32,
     objects: &mut Vec<BlockObject>,
     call: &pallet_feeds::Call<Runtime>,
-    successful_calls: &mut VecDeque<Hash>,
+    successful_calls: &mut Peekable<IntoIter<Hash>>,
```

Comment:
My thinking was to have iterator over a slice instead of specific `IntoIter` of a vector, so it works with anything that can be treated as a slice. But might be fine as is for now.